### PR TITLE
Added ability to submit Bloqade Hamiltonians to Braket

### PR DESCRIPTION
**Pull request by johnzl-777**
Allow conversion from `TaskSpecification` to the Braket IR as well as submitting Hamiltonians to Braket.

### CODE REVIEW

File: lib/BloqadeSchema/src/submit_to_braket.jl [hunk 1]
```diff
@@ -0,0 +1,100 @@
+using BloqadeSchema
+using BloqadeExpr
+using AWS
```
Comment (weinbe58) [inline lib/BloqadeSchema/src/submit_to_braket.jl:3]: Move these "using" statements to BloqadeSchema.jl

File: lib/BloqadeSchema/src/to_braket_ahs_ir.jl [hunk 2]
```diff
@@ -0,0 +1,151 @@
+using Braket
+using DecFP
```
Comment (weinbe58) [inline lib/BloqadeSchema/src/to_braket_ahs_ir.jl:2]: Same as above

**Review by weinbe58 — CHANGES_REQUESTED** [review]
Over all looks good one thing we should try to think about is how to test the submission to braket maybe we can rope in someone as aws.

**Comment by johnzl-777** [comment]
> Over all looks good one thing we should try to think about is how to test the submission to braket maybe we can rope in someone as aws.

Is this a blocker for getting this PR merged?

**Comment by weinbe58** [comment]
> > Over all looks good one thing we should try to think about is how to test the submission to braket maybe we can rope in someone as aws.
> 
> Is this a blocker for getting this PR merged?

No, we know it works we just don't have test coverage.

File: lib/BloqadeSchema/src/submit_to_braket.jl [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+
+
+# Keyword Arguments
+- `arn="arn:aws:braket:us-east-1::device/qpu/quera/Aquila"`: ARN for the machine
+- `region="us-east-1"`: AWS Region machine is located in
+- `credentials::Union{AWSCredentials, Nothing}=nothing`: `AWS.AWSCredentials` instance you can create to login.
+
+Credentials can be passed in explicitly through an `AWS.AWSCredentials` struct or by passing in 
+`nothing`, in which case credentials will be sought after in the shell environment variables.
```
Comment (weinbe58) [inline lib/BloqadeSchema/src/submit_to_braket.jl:15]:
```suggestion
`nothing`, in which case credentials will be sought in the standard AWS locations. 
```

File: lib/BloqadeSchema/src/submit_to_braket.jl [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+
+"""
+    submit_to_braket(h::BloqadeExpr.Hamiltonian, translation_params::BloqadeSchema.SchemaTranslationParams; <keyword arguments>)
+
+Submits a `BloqadeExpr.Hamiltonian` instance to Braket with `BloqadeSchema.SchemaTranslationParams` containing the number of shots and 
+device capabilities, returning an `AWS.AwsQuantumTask` and `Bloqade.HardwareTransformInfo` upon
+converting the Hamiltonian to one the hardware can execute.
+
+Credentials can be passed in explicitly through an `AWS.AWSCredentials` struct or by passing in 
+`nothing`, in which case credentials will be sought after in the shell environment variables.
```
Comment (weinbe58) [inline lib/BloqadeSchema/src/submit_to_braket.jl:45]:
```suggestion
`nothing`, in which case credentials will be sought in the standard AWS locations. 
```

File: lib/BloqadeSchema/src/submit_to_braket.jl [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+    return task, transform_info
+end
+
+"""
+    sumbit_to_braket(ts:BloqadeSchema.TaskSpecification; <keyword arguments>)
+
+Submits a `BloqadeSchema.TaskSpecification` instance to Braket, returning an `AWS.AwsQuantumTask`.
+
+Credentials can be passed in explicitly through an `AWS.AWSCredentials` struct or by passing in 
+`nothing`, in which case credentials will be sought after in the shell environment variables.
```
Comment (weinbe58) [inline lib/BloqadeSchema/src/submit_to_braket.jl:71]:
```suggestion
`nothing`, in which case credentials will be sought in the standard AWS locations. 
```

**Review by weinbe58 — APPROVED** [review]
After these changes you can merge.

**Pull request merged**